src/include/internal/error_handler.h

```diff
@@ -18,6 +18,11 @@
  */
 #ifndef ERROR_HANDLER_H_
 #define ERROR_HANDLER_H_
+
+/* User defined error codes set greater than unix error codes.*/
+#define INVALID_ETAG        133	/* Invalid ETag */
```

Comment:
It would be better to use enum here rather than #define
``` 
enum  {
          INVALID_ETAG = 133
          BAD_DIGEST = 134
          };
```

Reply:
sure, good idea

Reply:
done